akeyless_gateway/README.md

```diff
@@ -0,0 +1,104 @@
+
+This integration allows you to visualize and monitor performance of your [Akeyless Gateway](https://docs.akeyless.io/docs/api-gw). The Telemetry Metrics are based on time series telemetry data metrics from the application and the runtime environment, storing them in a unique database or index, and analyzing data trends over time.
+
+## Setup
+
+Akeyless offers a unique Gateway, which adds an extra level of protection between your private network and the cloud. Acting as a SaaS extension of our core services, our stateless Gateway enables a transparent internal operation with a robust out-of-the-box mechanism to ensure service continuity and recovery while you are not required to change any network infrastructure in order to work with your internal resources.
+
+To configure the integration with Datadog to view important Akeyless Gateway metrics, follow the below instructions for the method you are using (or have used) for your Gateway deployment.
+
+### Prerequisites
+- An Akeyless Gateway either running or being deployed for the first time
+
+### Configure
+
+This integration works with one Gateway or multiple instances using the same API key. Metrics can be shown per `host` or `instance` in the **Akeyless GW** dashboard.
+
+#### For a Gateway Running on Kubernetes
+
+To configure the Akeyless Gateway integration on a [Gateway running on K8s](https://docs.akeyless.io/docs/gateway-k8s):
+
+1. In your `values.yaml` file you use to deploy your Gateway on Kubernetes, under the `metrics` section, add the below configuration. Set the relevant API Key of your Datadog server, and the relevant site. If your Datadog server is running in the EU site, use `datadoghq.eu`. Default is `datadoghq.us`.
+
+```
+metrics:
+  enabled: true  
+  config: |
+    exporters:    
+      datadog:
+        api:
+          key: "<Your Datadog API key>"
+          site: <Your Datadog server site>         
+    service:
+      pipelines:
+        metrics:
+          exporters: [datadog]
+```
+
+2. If you have not yet deployed the Gateway, continue with your Installation as usual and run the following command when ready to deploy:
+
+```
+helm install <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+3. If you are updating an existing Gateway on Kubernetes, run the following commands to update:
+
+```
+helm upgrade <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+#### For a Standalone Gateway Running on Docker
+
+To configure the Akeyless Gateway integration on a [Standalone Gateway](https://docs.akeyless.io/docs/install-and-configure-the-gateway):
+
+1. Create a local file called `otel-config.yaml` with the below configuration. Set the relevant API Key of your Datadog server, and the relevant site. If your Datadog server is running in the EU site, use `datadoghq.eu`. Default is `datadoghq.us`.
```

Comment:
```suggestion
1. Create a local file called `otel-config.yaml` with the below configuration. Set the relevant API Key of your Datadog server, and the relevant Datadog site. 
```

Reply:
Same suggestion here as before to just leave the last sentence: "The default server is datadoghq.us."